package.json

```diff
@@ -204,6 +205,19 @@
       "crypto-js@<4.2.0": ">=4.2.0",
       "semver@>=7.0.0 <7.5.2": ">=7.5.2",
       "axios@>=0.8.1 <0.28.0": ">=0.28.0"
-    }
+    },
+    "onlyBuiltDependencies": [
+      "@parcel/watcher",
+      "@prisma/client",
+      "@prisma/engines",
+      "@swc/core",
+      "core-js",
+      "core-js-pure",
+      "cypress",
+      "esbuild",
+      "prisma",
+      "sharp",
+      "supabase"
+    ]
```

Comment:
new version of pnpm wants us to say if it's okay to run these during installation